pubtools/_pulp/hooks.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+from pubtools.pluggy import hookspec, pm
+
+
+@hookspec
+def task_pulp_flush():
+    """Invoked during task execution after successful completion of all
```

Comment:
Would you mind extending the doc string a little?  A concrete suggestion is below.

The main points are:
- the "all" Pulp writes unfortunately turns out to be false - looking at the code I can see now that we have to set cdn_published field after this hook is invoked.  Annoying, but the code is right so the doc string should be adjusted.
- probably good to say what's the intended usage of it (similarly as mentioned in the commit message)

So my attempt at a doc string would be:

```
Invoked during task execution after successful completion of all Pulp publishes.

This hook is invoked a maximum of once per task, to indicate that all Pulp content
associated with the task is considered fully up-to-date. The intended usage is to
flush Pulp-derived caches or to notify systems that Pulp content may have recently
changed.
```

By the way, this ends up at https://release-engineering.github.io/pubtools/hooks.html#hook-reference in case it wasn't clear.